client/a8c-for-agencies/sections/marketplace/checkout/notice-summary.tsx

```diff
@@ -0,0 +1,144 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useMemo } from 'react';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+type Props = {
+	type: 'client-purchase' | 'agency-purchase' | 'request-client-payment' | 'request-payment-method';
+};
+
+export default function NoticeSummary( { type }: Props ) {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const title = useMemo( () => {
+		switch ( type ) {
+			case 'client-purchase':
+			case 'agency-purchase':
+				return translate( 'When you purchase:' );
+			case 'request-client-payment':
+				return translate( 'When you send this payment request:' );
+			default:
+				return null;
+		}
+	}, [ translate, type ] );
+
+	const handleClientTermsClick = useCallback( () => {
+		dispatch( recordTracksEvent( 'calypso_a4a_client_checkout_client_terms_click' ) );
+	}, [ dispatch ] );
+
+	const handleAgencyTermsClick = useCallback( () => {
+		dispatch( recordTracksEvent( 'calypso_a4a_client_checkout_agency_terms_click' ) );
+	}, [ dispatch ] );
+
+	const handleSubscriptionInfoLinkClick = useCallback( () => {
+		dispatch( recordTracksEvent( 'calypso_a4a_client_checkout_subscription_info_link_click' ) );
+	}, [ dispatch ] );
+
+	const handleCancellationInfoLinkClick = useCallback( () => {
+		dispatch( recordTracksEvent( 'calypso_a4a_client_checkout_cancellation_info_link_click' ) );
+	}, [ dispatch ] );
+
+	const items = useMemo( () => {
+		switch ( type ) {
+			case 'client-purchase':
+			case 'agency-purchase':
+				return [
+					translate(
+						'You agree to our {{a/}}, and authorize your payment method to be charged on a recurring basis until you cancel, which you can do at any time.',
+						{
+							components: {
+								a:
+									type === 'client-purchase' ? (
+										<a
+											href={ localizeUrl( 'https://wordpress.com/tos' ) }
+											onClick={ handleClientTermsClick }
+											target="_blank"
+											rel="noreferrer noopener"
+										>
+											{ translate( 'Terms of Service' ) }
+										</a>
+									) : (
+										<a
+											href={ localizeUrl(
+												'https://automattic.com/for-agencies/platform-agreement/'
+											) }
+											onClick={ handleAgencyTermsClick }
+											target="_blank"
+											rel="noreferrer noopener"
+										>
+											{ translate( 'Platform agreement' ) }
```

Comment:
I think even this has to be `Terms of Service` as per the design and should link to https://wordpress.com/tos. Am I missing something here?

Reply:
I believe the design missed this part. I was referring on the p2 discussion.

pfunGA-1rU-p2#comment-3562

Reply:
Cc @jeffgolenski can you confirm which one is correct?

Reply:
@yashwin @jkguidaven Agencies agree to the Platform Agreement when they sign up. This link during checkout, should be a link to the TOS that Jenny mentions here: pfunGA-1rU-p2#comment-3540

Reply:
I see. Thanks for confirming. I'll make the changes to fix this.

Reply:
This has been addressed.